src/nearest_neighbors/KozachenkoLeonenko.jl

```diff
@@ -0,0 +1,36 @@
+export KozachenkoLeonenko, genentropy
+
+"""
+## Nearest neighbour(NN) based
+
+    KozachenkoLeonenko(; w::Int = 1) <: NearestNeighborEntropyEstimator
+
+Entropy estimator based on nearest neighbors. This implementation is based on Kozachenko & Leonenko (1987)[^KozachenkoLeonenko1987],
+as described in Charzyńska and Gambin (2016)[^Charzyńska2016].
+
+`w` is the number of nearest neighbors to exclude when searching for neighbours 
+(defaults to `0`, meaning that only the point itself is excluded).
```

Comment:
It's simpler to say here that `w` is just the Theiler window.

Reply:
Agreed. The docstring mentions the Theiler window now.